RecoLocalCalo/EcalRecAlgos/interface/EigenMatrixTypes.h

```diff
@@ -1,16 +1,30 @@
+#ifndef EigenMatrixTypes_h
+#define EigenMatrixTypes_h
```

Comment:
Yes ok I'll leave them here and just change the ifndef as suggested.
The actual numbers which are here are pretty detector and algorithm specific, so I think it makes sense to keep them here anyways.